src/vs/workbench/contrib/debug/node/debugAdapter.ts

```diff
@@ -441,7 +440,7 @@ export class ExecutableDebugAdapter extends StreamDebugAdapter {
 		const result: IDebuggerContribution = Object.create(null);
 		if (contribution.runtime) {
 			if (contribution.runtime.indexOf('./') === 0) {	// TODO
-				result.runtime = extpath.join(extensionFolderPath, contribution.runtime);
+				result.runtime = path.join(extensionFolderPath, contribution.runtime);
```

Comment:
@weinand fyi as discussed this changes to backslash now. Please consider revisiting the TODO in the line above, maybe use `path.isAbsolute()` I tried that but it resulted in a test failure?